go.sum

```diff
@@ -1,15 +1,14 @@
-github.com/amzn/ion-go v1.1.0 h1:699A2uZDtkPf5yTMcRkTMoCBYa0UELLgAt0k8Psw+RY=
-github.com/amzn/ion-go v1.1.0/go.mod h1:93Bu1K0O/CDosTCDzJ1cNY7XpdHGAGE2NepHcGsiV70=
-github.com/amzn/ion-hash-go v1.1.0 h1:m7UeSm77kENRJCI1IbcRVRZNwh4/Et2uQoFPf+1x5e0=
-github.com/amzn/ion-hash-go v1.1.0/go.mod h1:UF0NLRWtcpfgBneeqK0ESnN7iSmqpRvuY9VHvD4XwNg=
+github.com/amzn/ion-go v1.1.2/go.mod h1:7wQBWQ7PhPpZCr9PL+mtuIyNmyLjuV8qt2mrfxmvkA8=
```

Comment:
could you run `go mod tidy` ?

Reply:
I ran the command but it made no difference.